docs/clustering/multi-node-setup.rst

```diff
@@ -314,29 +315,26 @@ ensures that the cluster does not elect multiple masters in the event of a
 network partition (also known as a `split-brain`_ scenario).
 
 CrateDB (versions 4.x and above) will automatically determine the ideal `quorum
-size`_. If you are using CrateDB versions 3.x and below, you must manually set
-the quorum size using the `discovery.zen.minimum_master_nodes`_ setting.
-
-.. NOTE::
-
-    For a three-node cluster, CrateDB will set the quorum size to three.
-    Consequentially, you must declare all nodes to be master-eligible. Consult
-    the `quorum guide`_ for detailed information about quorum size
-    considerations.
-
-If you configure fewer master-eligible nodes than the ideal quorum size,
-CrateDB will issue a warning (visible in the logs and the `admin UI`_).
-
+size`_, but if you are using CrateDB versions 3.x and below, you must manually set
+the quorum size using the `discovery.zen.minimum_master_nodes`_ setting and for 
+a three-node cluster, you must declare all nodes to be master-eligible. 
 
 .. _metadata-gateway:
 
 Metadata gateway
 ^^^^^^^^^^^^^^^^
 
-When running a multi-node cluster, you must configure the `metadata gateway`_
-so that CrateDB knows how to recover its state. For a three-node cluster, set
-`gateway.expected_data_nodes`_ to **3** and `gateway.recover_after_data_nodes`_
-to **3**.
+When running a multi-node cluster, you can configure the `metadata gateway`_
+settings so that CrateDB delays recovery until a certain number of nodes is
+available.
+This is useful because if recovery is started when some nodes are down 
+CrateDB will proceed on the basis the nodes that are down may not be coming 
+back, and it will create new replicas and rebalance shards as necessary, 
+this is an expensive operation that, depending on the context, may be better 
```

Comment:
Spotted a little chance for improving the reading flow.
```suggestion
back, and it will create new replicas and rebalance shards as necessary. 
This is an expensive operation that, depending on the context, may be better 
```